examples/plot_firing_pattern.py

```diff
@@ -54,24 +55,15 @@
 plt.show()
 
 ###############################################################################
-# Let's do this for the rest of the cell types
-fig, axes = plt.subplots(1, 2, sharey=True, figsize=(8, 4))
-for gid, ax in zip([gid_dict['L2_pyramidal'][0],
-                    gid_dict['L5_pyramidal'][0]], axes):
-    ax.plot(times, vsoma[gid])
-    ax.set_title('%s (gid=%d)' % (net.gid_to_type(gid), gid))
```

Comment:
It'd be nice to keep an example of `gid_to_type()`, unless it's used in another example?

Reply:
It's actually above: https://github.com/jonescompneurolab/hnn-core/pull/206/files#diff-a39b3b79040c9c527504aea86494f6d09a2716ff1b130fdc3845d5485b7d48e6R52

Reply:
Ah, hidden for people too lazy to check the original.

Yeah this looks much more concise and clean, definitely merge unless you've got other ideas to tack on.

Reply:
also the colors match between different plots and the time axis is common. That was the main purpose :)

go ahead and merge if you're happy!